include/boost/geometry/strategies/intersection_result.hpp

```diff
@@ -26,7 +31,7 @@ namespace boost { namespace geometry
     \ingroup overlay
     \see http://gis.hsr.ch/wiki/images/3/3d/9dem_springer.pdf
 */
-struct de9im
+/*struct de9im
```

Comment:
Yes, I think it could be removed though I'd like to hear from Barend what I should do with it because AFAIU he was the one who contributed this code.